components/ListView/ComponentInputs.js

```diff
@@ -1,16 +1,16 @@
 /* global $ */
 
 import React from 'react';
-import {defineMessages, injectIntl, intlShape} from 'react-intl';
+import { defineMessages, injectIntl, intlShape } from 'react-intl';
```

Comment:
There are a lot of these formatting changes in this PR. Would be nice if they could be done in a separate commit, as this makes it hard to review what actually changed.